CMakeLists.txt

```diff
@@ -241,6 +279,37 @@ if(NOT NONET)
   target_link_libraries(devilutionx PRIVATE sodium)
 endif()
 
+if(USE_SDL1)
+  target_link_libraries(devilutionx PRIVATE
+    ${SDL_TTF_LIBRARY}
+    ${SDL_MIXER_LIBRARY}
+    ${SDL_MIXER_LIBRARIES}
+    ${SDL_LIBRARY}
+    ${FREETYPE_LIBRARIES}
+    ${PNG_LIBRARIES})
+  if(WARPOS)
+    target_link_libraries(devilutionx PRIVATE
+      -lmikmod
+      -lvorbisfile

```

Comment:
vorbisfile, vorbisenc, vorbis, ogg, FLAC and bz2 not needed

Reply:
SDl_Mixer needs it.

Reply:
It doesn't need it if it isn't compiled with the support for it.

Reply:
It isn't specifically required by DevilutionX